code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1152,13 +1153,13 @@
 	if((temp_turf.z != our_turf.z) || M.stat!=CONSCIOUS) //Not on the same zlevel as us or they're dead.
 //		to_chat(world, "[(temp_turf.z != our_turf.z) ? "not on the same zlevel as [M]" : "[M] is not concious"]")
 		if(temp_turf.z != 2)
-			to_chat(src, "The mind of [M] is too faint...")//Prevent "The mind of Admin is too faint..."
-
-
+			if(prob(5))//Can't tell who is missing as easily as before. I feel like this is a fun alternative to removing it altogether
```

Comment:
I mean player feedback from a gameplay perspective. As in, the way the game gives a response to the player doing something. Think construction sounds, action messages, etc.